tests/fuzz/fuzz_common.h

```diff
@@ -0,0 +1,62 @@
+//
+// Created by profanter on 18.07.17.
+// Copyright (c) 2017 fortiss GmbH. All rights reserved.
+//
+
+#ifndef OPEN62541_FUZZ_COMMON_H_H
```

Comment:
Is it not possible to simply replace ua_network_tcp.c with testing_networklayer.c in fuzzing_plugin_sources?